extensions/lifecycle/bucketProcessor/LifecycleBucketProcessor.js

```diff
@@ -340,6 +508,9 @@ class LifecycleBucketProcessor {
             done => this._setupProducer(done),
             done => this._initKafkaBacklogMetrics(done),
             done => this._setupConsumer(done),
+            done => this._setupMongoClient(done),
+            done => this._initializeLocationStatuses(done),
```

Comment:
there is a race condition here:
* Read initial state → paused
* User triggers a resume, which is added to mongo & all...
* Create the change stream → no event received
* Processor is still paused...

we should first setup the change stream, then look for the initial state ; and in the change stream callback we need to be resilient to both:
* being the first event, e.g. happening before we could do the initial state management
* being already in the expected state
(should not be hard, but just to keep in ming)